.gitignore

```diff
@@ -13,5 +13,5 @@ TODO
 _docs
 examples/winds/data/*.html
 node_modules
-npm-debug.log
+*.log*
```

Comment:
Hum yeah `npm-debug.log` sometimes gets suffixed with numbers, maybe `*.log.*` would be better.